packages/amplify-migration-tests/src/__tests__/migration_tests/lambda-layer-migration/layer-migration.test.ts

```diff
@@ -1,6 +1,6 @@
 import {
   addFunction,
-  amplifyPushAuth,
+  amplifyPushAuthLegacy,
```

Comment:
instead of `Legacy` can we name this with the version of amplify that the function is meant to be executed with?

Reply:
updated, even though this would for any version that will be before this change gets released it is used for migrations from V5 and V6 to latest